examples/library/test_struct_size/main.c

```diff
@@ -36,6 +35,7 @@
 #include <bsg_manycore_printing.h>
 #include <cl_manycore_regression.h>
 
+// This test seems to be superseded by test_packet. Do we need it?
```

Comment:
Not sure how @drichmond feels about it, but my preference for PRs is to add things like this as GH comments rather than code comments, and then reviewers can determine whether to open another issue/PR. Otherwise they tend to get lost in the noise

Reply:
That makes sense. So I reverted to using GH comments.